pypeit/wavecalib.py

```diff
@@ -358,7 +360,7 @@ class BuildWaveCalib:
 
     frametype = 'wv_calib'
 
-    def __init__(self, msarc, slits, spectrograph, par, lamps, binspectral=None, det=1,
+    def __init__(self, msarc, slits, spectrograph, par, lamps, binspectral=None, meta_dict=None, det=1,
```

Comment:
Add to the doc string, and describe what's in the dict that's needed by the code.

Reply:
Added.